### ISSUE TITLE
Multimonitor (2x 40" + 2x 23") - Black screen if connected via HDMI - works via DP

### ISSUE THREAD

**Issue by andreas-it-dev** [linuxmint/cinnamon-control-center#221]
```
 * Cinnamon version (cinnamon --version)
   4.2.2
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
   Manjaro
 * Graphics hardware *and* driver used
https://termbin.com/m0m9
 * 32 or 64 bit
   64bit
 * Attach /home/<username>/.xsession-errors, or /var/log/syslog
[xsession-errors.txt](https://github.com/linuxmint/cinnamon/files/3388896/xsession-errors.txt)

 ```

**Issue**
when i connect my 4th monitor via hdmi it does not light up when used with full resolution (3840x2160). it does when i only have it using 2560x1440.
it does work entirely when i connect it via disply port though that has its own issues (moved windows after powering off the monitor / when it went to standby)

![DeepinScreenshot_nemo-desktop_20190713103617](https://user-images.githubusercontent.com/13486613/61173014-3caba680-a55b-11e9-989e-7206a6f1c4eb.png)

**Steps to reproduce**
for me this is just to plug it in

**Expected behaviour**
the monitor should work with hdmi

**Other information**
cinnamon thinks though that the monitor is connected and working as it lets me drag windows over onto it and it also "shows" windows on the screen.. however, the monitor itself shuts down to standby reporting no activity

this entire setup incl. hdmi works under KDE

**Comment by awunder** [comment]
fwiw, gnome has the same issue though i can somewhat cheat me through to get it working when i set the resolution to 2560x1440 and then back to 3840x2160..

that same trick does not work under cinnamon

**Comment by icarter09** [comment]
@awunder are you still experiencing this issue?

**Comment by awunder** [comment]
@icarter09 yes, i do.. i somewhat figured out a workaround though.. it seems to be related to the refresh rate.. it sets it to 29.96Hz after setting the resolution and my monitor doesnt like that.
what i figured out was when i set the rate back to 30 via xrandr it lights up again.

so i created me a little script and start that at every reboot:

```
#!/bin/bash
xrandr --output HDMI-0 --mode 3840x2160 --rate 30
xrandr --output HDMI-1-1 --mode 3840x2160 --rate 30
```

that does the trick for now.. but its still only a workaround, imho.. so, a fix or at least some form of refresh rate picker (like gnome or kde has) is still appreciated.
